### DISCUSSION TITLE
Apply border opacity to individual side

### DISCUSSION THREAD

**Discussion by StinkyDubeau** [tailwindlabs/tailwindcss#13084] [Ideas]
*(My first discussion, sorry if it sucks)*

It would be nice if, much like `border-t-red`, we could apply opacity to specific sides of an element. I.e. `border-t-opacity-25`.

I can accomplish the same thing with custom colours, `border-t-[#ffffff33]`, but it would be nice if the above syntax worked.

Thanks

**Comment by alex-krasikau** [comment]
Hey @StinkyDubeau!

Tailwind CSS has an additional color opacity modifier. The solution is `border-t-red-500/50`. You can see a full example here: [https://play.tailwindcss.com/P54VBk5LdW](https://play.tailwindcss.com/P54VBk5LdW). For more details, check out the [Border Opacity](https://tailwindcss.com/docs/border-color#changing-the-opacity) documentation.
